RedHat-UX/red-hat-design-system
# docs: update screenshot files for icon and health index

## What I did

1. Updated the `screenshot.png` files for both `rh-icon` and `rh-health-index` to be the appropriate dimensions (680x400).


## Testing Instructions

1. Visit the DP.
2. Go to Elements > [All Elements](https://deploy-preview-1799--red-hat-design-system.netlify.app/elements/)
3. Make sure the screenshots for `rh-icon` and `rh-health-index` are the same height and look like the rest of the icons on the page.
